types/heremaps/index.d.ts

```diff
@@ -5681,7 +5694,7 @@ declare namespace H {
              * @param opt_locale {(H.ui.i18n.Localization | string)=} - the language to use (or a full localization object).
              * @returns {H.ui.UI} - the UI instance configured with the default controls
              */
-            static createDefault(map: H.Map, mapTypes: H.service.Platform.MapTypes, opt_locale?: H.ui.i18n.Localization | string): UI;
+            static createDefault(map: H.Map, mapTypes: H.service.MapType | H.service.DefaultLayers, opt_locale?: H.ui.i18n.Localization | string): H.ui.UI;
```

Comment:
After reviewing the documentation, it doesn't seem like `H.service.MapType` would be a valid argument for the `mapTypes` parameter. Should this have been `H.service.Platform.MapTypes | H.service.DefaultLayers`?

Also, it seems like `H.service.Platform.MapTypes` already defines optional members for several of the members in the new `H.service.DefaultLayers`, as well as an indexer to allow additional map types. Would it make more sense just to add the remaining members to `MapTypes` as optional members?